src/components/Content/ContentList.css

```diff
@@ -4,34 +4,25 @@
   @mixin remove-a;
 }
 
-.element {
-  height: 50px;
-}
-
-.element-text {
-  height: 100%;
-  min-width: 1px; /* exact value doesn't matter, but this lets long words wrap */
-  flex-grow: 1;
-  margin-left: 15px;
-  padding: 15px 0;
-  border-bottom: 2px solid #fbda16;
-  word-wrap: break-word;
-}
-
 .row {
   display: flex;
   align-items: center;
   margin-bottom: 10px;
 }
 
-.element-image {
+.categoryThumbnail {
   width: 50px;
+  height: 50px;
   flex-shrink: 0;
   padding: 5px;
 }
 
-.image {
-  width: 50px;
-  max-width: 100%;
-  max-height: 100%;
+.categoryCaption {
+  height: 100%;
+  min-width: 1px; /* needed to enabled line breaks for to long words, exact value doesn't matter, @Max: DO NOT CHANGE */
+  flex-grow: 1;
+  margin-left: 15px;
+  padding: 15px 0;
+  border-bottom: 2px solid #fbda16;
```

Comment:
Is there a difference between #fafafa and #fbda16? Not sure if we decided on I9K. If we know for sure we can also create a variable in a global Colors.css somewhere.

Reply:
i think we always use #fbda16 except from the background color of the header in RichLayout

Reply:
@maxammann #fafafa is grey, #fbda16 is integreat-yellow. Colors.css will be introduced in https://integreat.atlassian.net/browse/WEBAPP-106

Reply:
alright :+1:  

our new grey is #efefef, if you have time you can change it right here

Reply:
i think we should change all colors with WEBAPP-106, so that our design is consistent